packages/session-ui/src/components/message-part.css

```
@@ -138,29 +138,43 @@
     align-items: flex-end;
   }
 
+  /* The prompt bubble, from the website's /amicode chat (harmoniqs-ai
+     app/components/demo/parts.jsx, `Turn`): the user's words are the INVERSE
+     of the ground — bg-deep/text-bg there. bg-inverse/text-inverse is that
+     same pair in both schemes: an ink bubble on light, a near-white bubble on
+     dark. Padding is the site's px-4 py-2.5. */
   [data-slot="user-message-text"] {
     display: inline-block;
     white-space: pre-wrap;
     word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)